apps/dataset-browser/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "tsconfig/nextjs.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
+    }
+  },
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
```

Comment:
Curious: `next-env.d.ts` is already set in the parent tsconfig. Is it automatically used or must we explicitly set it again here?

Reply:
We can remove this